### ISSUE TITLE
icon color contrast insufficient

### ISSUE THREAD

**Issue by csps-snap** [DIS-SIN/Busrides#86]
The busrude icon in the navigation at the top has insufficient contrast.

![image](https://user-images.githubusercontent.com/41083520/138285503-70af0706-1986-4000-98b7-882cefc4c0cd.png)

suggesting using white instead.
![image](https://user-images.githubusercontent.com/41083520/138285311-d0c34818-4a12-4485-b754-ea404f3105ea.png)

**Comment by romy-piche** [comment]
The icon has been replaced by the white one
